src/main/java/org/opentripplanner/middleware/utils/HttpUtils.java

```diff
@@ -147,7 +148,7 @@ public static HttpResponseValues httpRequestRawResponse(URI uri, int timeoutInSe
         try  {
             // Extract required information from the response and return to caller. The connection is closed once complete.
             return httpClient.execute(httpUriRequest, new HttpResponseHandler(httpUriRequest));
-        } catch (HttpTimeoutException e) {
+        } catch (HttpTimeoutException | SocketTimeoutException e) {
```

Comment:
Do you have a proposal for how we are alerted about request timeouts? It feels kind of sketchy to just ignore timeouts, given that they alerted us to some major issues with our ATL implementation of OTP.